internal/controller/driver_controller.go

```diff
@@ -477,6 +479,9 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 										utils.CsiAddonsEndpointContainerArg,
 										"",
 									),
+									utils.If(r.logRotation(), utils.LogRotationSpecArg(r.driver.Spec.Log.Rotation.LogHostPath, deploy.Name, fmt.Sprintf("csi-%splugin", r.driverType)), ""),
+									utils.If(r.logRotation(), utils.LogToFile, ""),
+									utils.If(r.logRotation(), utils.LogToStderr, ""),
```

Comment:
These three arguments are repeated in many containers. Please calc them into a separate Array at the op of the function, where you can start with an empty slice then append if log rotation is enabled, then join/concat them into the existing argument list